src/main/java/Duke.java

```diff
@@ -2,30 +2,109 @@
+        System.out.println("    Bye. Hope to see you again soon!");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printHello() {
         System.out.println("____________________________________________________________");
         System.out.println("    Hi! I'm Duke (:");
         System.out.println("    What can I do for you today?");
         System.out.println("____________________________________________________________");
-        command = in.nextLine();
-        do {
-            if (command.equals("list")) {
-                Task.listTask();
-            }
-            else {
-                if (command.contains("done")) {
-                    Task.doneTask(command);
-                }
-                else {
-                    Task.addTask(command);
-                }
+    }
+
+    public static void printDone(int description) {
+        System.out.println("____________________________________________________________");
+        System.out.println("This task has been done. Good job!");
+        tasks[description].printDescription();
+        System.out.println("\n" + "____________________________________________________________");
+    }
+
+    public static void printTotalTasks() {
+        if (numberOfTasks >= 2) {
+            System.out.println(numberOfTasks + " tasks left in the list");
+        } else if (numberOfTasks == 1) {
+            System.out.println("Only 1 task in the list");
+        }
+    }
+
+    public static void listTask() {
+        for (int i = 1; i <= numberOfTasks; ++i) {
+            System.out.print(i + ".");
+            switch (keyword[i - 1]) {
+            case "T":
+                System.out.print("[T]");
+                tasks[i - 1].printDescription();
+                System.out.print("\n");
+                break;
+            case "D":
+                System.out.print("[D]");
+                tasks[i - 1].printDescription();
+                System.out.println("(by:" + by + ")");
+                break;
+            case "E":
+                System.out.print("[E]");
+                tasks[i - 1].printDescription();
+                System.out.println("(at:" + at + ")");
+                break;
             }
+        }
+    }
+
+        public static void main (String[]args){
+            String command;
+            String[] words;
+
+            Scanner in = new Scanner(System.in);
+            printHello();
             command = in.nextLine();
-        } while (!command.equals("bye"));
-        System.out.println("____________________________________________________________");
-        System.out.println("    Bye. Hope to see you again soon!");
-        System.out.println("____________________________________________________________");
+            do {
+                if (command.equals("list")) {
+                    listTask();
+                } else {
+                    if (command.contains("done")) {
```

Comment:
consider using a switch case statement! but also i like how this should work as long as the command word is in the text!